browser/ui/views/tabs/brave_compound_tab_container.cc

```diff
@@ -24,6 +25,10 @@
 #include "ui/views/layout/flex_layout.h"
 #include "ui/views/view_utils.h"
 
+#if !BUILDFLAG(IS_MAC)
+#include "ui/views/controls/scrollbar/scroll_bar_views.h"
```

Comment:
Our preferred way is aligning build configuration between gn and c++.
so this including doesn't need to be in `IS_MAC` buildflag in this file

Reply:
Good call. Done.